src/lexfloatclient.ts

```diff
@@ -127,6 +135,25 @@ export class LexFloatClient {
 			throw new LexFloatClientException(status);
 		}
 	}
+	
+	/**
+	 * Sets the permission flag for your application.
+	 *
+	 * This function must be called on every start of your program after the SetHostProductId()
+	 * function if the application allows borrowing of licenses or system-wide activation.
+	 *
+	 * @param {PermissionFlags} flag - depending on your application's requirements, choose one of 
+	 * the following values: LF_USER, LF_ALL_USERS.
+	 *  - LF_USER: This flag indicates that the application does not require admin or root permissions to run.
+	 *  - LF_ALL_USERS: This flag is specifically designed for Windows and should be used for system-wide activations.
+	 * @throws {LexFloatClientException} If the status code is not LF_OK or LF_E_PRODUCT_ID.
+	 */
```

Comment:
@muneebkq Update the docs, it should be  `@throws {LexFloatClientException} . Please check how the docs for the exceptions are for other functions.

Additionally, Is this correct based on what the code says?

Reply:
jsdoc modified. Rest everthing is correct as per code